src/mixins/routable.ts

```diff
@@ -39,8 +45,8 @@ export default Vue.extend({
         props: {},
         directives: [{
           name: 'ripple',
-          value: (this.ripple && !this.disabled) ? this.ripple : false
-        }] as any, // TODO
+          value: this.computedRipple
+        }] as any,
```

Comment:
I should have left a better comment, the TODO is to remove `as any` when vuejs/vue#8013 is fixed